src/main/steps/applicant1/nationality/content.ts

```diff
@@ -0,0 +1,139 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent, FormFieldsFn } from '../../../app/form/Form';
+import { atLeastOneFieldIsChecked, isFieldFilledIn } from '../../../app/form/validation';
+import { mapSummaryListContent } from '../../common/functions/mapSummaryListContent';
+import { APPLICANT_1_NATIONALITY } from '../../urls';
+
+export const en = (): Record<string, unknown> => ({
+  section: 'Primary applicant',
+  title: 'What is your nationality?',
+  label: 'Select all options that are relevant to you.',
+  british: 'British',
+  britishSubtext: 'including English, Scottish, Welsh and Northern Irish',
+  irish: 'Irish',
+  differentCountry: 'Citizen of a different country',
+  applicant1Nationality: 'Country name',
+  add: 'Add',
+  another: 'Add another country',
+  errors: {
+    nationality: {
+      required: 'Select if you are British, Irish or a citizen of a different country',
+    },
+    applicant1Nationality: {
+      required: 'This is not a valid entry',
+    },
```

Comment:
there is a validator on `addAnotherNationality` field hence you need `addAnotherNationality` `required` error message here